src/test/groovy/com/gooddata/executeafm/result/ResultHeaderItemTest.groovy

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.executeafm.result
+
+import spock.lang.Specification
+import spock.lang.Unroll
+
+import static com.gooddata.util.ResourceUtils.readObjectFromResource
+
+class ResultHeaderItemTest extends Specification {
```

Comment:
What about some sort of integration test extension?

Reply:
thare is no Service yet, so no IT to extend ....